test/e2e/performanceprofile/functests/10_performance_ppc/ppc.go

```diff
@@ -193,5 +193,65 @@ var _ = Describe("[rfe_id: 38968] PerformanceProfile setup helper and platform a
 			Eventually(session).Should(gexec.Exit(1))
 		})
 
+		It("[test_id: 54187] PPC generates profile with PerPodPowerManagement workload hint", func() {
+			defaultArgs := []string{
+				"run",
+				"--entrypoint",
+				"performance-profile-creator",
+				"-v",
+			}
+			pp := &performancev2.PerformanceProfile{}
+			cmdArgs := []string{
+				fmt.Sprintf("%s:%s:z", mustgatherDir, mustgatherDir),
+				ntoImage,
+				fmt.Sprintf("--mcp-name=%s", mcpName),
+				fmt.Sprintf("--reserved-cpu-count=%d", 4),
```

Comment:
it's a consts so 
`"--reserved-cpu-count=4"` is good enough

Reply:
fixed in the latest commit